clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -252,7 +252,13 @@ export class CopybookDownloadService {
       ...DialectRegistry.getActiveDialects().map((di) => di.name),
     ];
 
+    const copybooks: string[] = [];
+
+    const dsnPaths: string[] = SettingsService.getDsnPath(documentUri, dialect);
+    const ussPaths: string[] = SettingsService.getUssPath(documentUri, dialect);
+
     if (
+      (dsnPaths.length > 0 || ussPaths.length > 0) &&
       !(await this.isPrerequisiteForDownloadSatisfied(documentUri, dialects))
```

Comment:
Shouldn't this just return early?
```suggestion
    const dsnPaths: string[] = SettingsService.getDsnPath(documentUri, dialect);
    const ussPaths: string[] = SettingsService.getUssPath(documentUri, dialect);
    
    if (dsnPaths.length === 0 && ussPaths.length === 0) {
      return [];
    }

    if (
      !(await this.isPrerequisiteForDownloadSatisfied(documentUri, dialects))
```